doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -30,6 +30,55 @@ If multiple drivers are enabled, the first ordered item in this table takes prec
 
 Enabling or disabling PSA driver specific configurations controls the support for a given algorithm, per driver.
 
+Key type configuration
+**********************
+
+To enable block or stream cipher algorithms, set one or more of the following Kconfig options:
```

Comment:
AFAIK RSA does not use a block or stream cipher.

Perhaps we should describe this in a different way:

Algorithms use keys with some flexibility in which keys can be used for which operations.
But to use a PSA key it's corresponding Kconfig option must first be enabled.

The Kconfig options follow a specific naming pattern: the PSA key PSA_KEY_TYPE_<type> is enabled by the Kconfig option CONFIG_PSA_WANT_KEY_TYPE_<type>.

For information about which keys can be used with specific algorithms, please refer to the PSA Crypto API documentation